src/_guides/language/language-tour.md

```diff
@@ -1723,33 +1723,38 @@ assert(foo() == null);
 ## Operators
 
 Dart supports the operators shown in the following table.
+The table shows Dart's operator associativity 
+and [operator precedence](#operator-precedence-example) from highest to lowest,
+which are an **approximation** of Dart's operator relationships.
 You can implement many of these [operators as class members](#_operators).
```

Comment:
**suggestion:** To tighten language, sometimes you need to write more, shorter sentences.
```suggestion
The following table lists the operators that Dart supports.
The table sorts the operators from highest to lowest precedence.
When determining which operator to apply first, Dart compares operators.
When comparing two different operators, 
Dart applies the operator with greater precedence first. 
When comparing two of the same operator, 
Dart applies the operator from the associative direction first. 
Complex expressions may apply operators in a recursive fashion. 
Due to the complexity of [operator-precedence grammar](https://en.wikipedia.org/wiki/Operator-precedence_grammar), use this table as a guideline only.  

You can use many of these operators as class members.
```

- [x] How it should only be used as a "helpful guide" not definitive
- [x] Briefly explaining why it can only be used as a helpful guide
- [x] Where/how to find and understand the authoritative behavior

Reply:
I can't help commenting here! ;-)

This doesn't tell the reader that the whole concept of operator precedence is imprecise. It's not a matter of complexity, there's just no reason to assume that there exists an operator precedence table which is a correct description of the behavior of a Dart parser. It would certainly be a non-trivial effort to establish any firm results in this area.

Reply:
I'm also a bit worried about providing so much information in the opening here, when most readers are just looking for the table. We expand on the the precedence right after the table already, and I like your comments around that, so I'll try to incorporate some of that there. I'm going to go ahead and open a new PR which does a larger overhaul of this operator section, incorporating some of everyone's thoughts.